docs/guides/getting-started/index.html

```diff
@@ -0,0 +1,81 @@
+---
+title: Getting Started
+---
+{% extends 'guide.njk' %}
+{% block content %}
+  <section>
+    <p>
+      This guide is intented to help developers initialize a new application.
```

Comment:
Wondering about the word 'new' and if it's necessary. Would a developer ever initialize an existing application? If not, you can remove the word 'new'.